### PR TITLE
319 workshop choice bar

### PR DESCRIPTION
**Pull request by radscheit**
(#319)

### CODE REVIEW

File: app/assets/stylesheets/requests.css [hunk 1]
```diff
@@ -2,3 +2,62 @@
   Place all the styles related to the matching controller here.
   They will automatically be included in application.css.
 */
+
+.owl-theme .owl-controls{
+margin-top:10px;
```
Comment (pinussilvestrus) [inline app/assets/stylesheets/requests.css]: can you add tabs in this stylesheet?

File: config/locales/de.requests.yml [hunk 2]
```diff
@@ -2,6 +2,43 @@
 
 de:
   requests:
+    templates:
+      headline: "Beispiel-Anfragen"
+      lead_paragraph: "Lorem ipsum dolor sit amet, consectetur adipisicing elit. Laborum fugiat quasi expedita distinctio, suscipit, eaque deleniti similique eveniet in accusantium voluptas blanditiis id modi voluptatibus culpa illum aliquam. Neque, dolore."
```
Comment (pinussilvestrus) [inline config/locales/de.requests.yml:7]: I don't know whether there is another story for it, but wouldn't it be better if here is a better content than Lorem ipsum?

**Review by pinussilvestrus — CHANGES_REQUESTED** [review]
image

this doesn't look correct or?

File: config/locales/de.requests.yml (hunk 2 above)
Comment (radscheit) [reply config/locales/de.requests.yml:7]: Done.

**Review by pinussilvestrus — APPROVED** [review]

**Pull request merged**